mailpoet/tests/acceptance/Newsletters/SaveNewsletterAsDraftCest.php

```diff
@@ -5,10 +5,40 @@
 class SaveNewsletterAsDraftCest {
   public function saveStandardNewsletterAsDraft(\AcceptanceTester $i) {
     $i->wantTo('Create standard newsletter and save as a draft');
+    $newsletterTitle = 'Testing Newsletter ' . \MailPoet\Util\Security::generateRandomString();
 
+    $segmentName = $i->createListWithSubscriber();
     $inactiveNewsletterUI = '.mailpoet-listing-row-inactive';
+
+    $this->startCreatingNewsletter($i, $newsletterTitle);
+
+    // step 4 - Choose list and send
+    $sendFormElement = '[data-automation-id="newsletter_send_form"]';
+    $i->waitForElement($sendFormElement);
+    $i->selectOptionInSelect2($segmentName);
+    $i->click('Save as draft and close');
+    $i->waitForText($newsletterTitle);
+    $this->assertNewsletterNotSent($i);
+  }
+
+  public function saveStandardNewsletterWithoutListAsDraft(\AcceptanceTester $i) {
+    $i->wantTo('Create standard newsletter and save as a draft without selecting subscribers list');
     $newsletterTitle = 'Testing Newsletter ' . \MailPoet\Util\Security::generateRandomString();
-    $segmentName = $i->createListWithSubscriber();
+
+    $this->startCreatingNewsletter($i, $newsletterTitle);
+
+    // step 4 - Choose list and send
```

Comment:
Sorry for the nitpick, but I guess we need to update this comment as the code below doesn´t choose a list.

Reply:
Updated the comment [in the last commit](https://github.com/mailpoet/mailpoet/pull/4328/commits/a5a75b4bb7ef71b7f5f809fb3f2ea39cd8aa9dff)